awesomeConfig.cmake

```diff
@@ -340,6 +340,18 @@ foreach(file ${AWESOME_CONFIGURE_FILES})
 endforeach()
 #}}}
 
+# {{{ Generate some aggregated documentation from lua script
+execute_process(
+        COMMAND lua ${SOURCE_DIR}/docs/06-appearance.md.lua
+        ${BUILD_DIR}/docs/06-appearance.md
+        ERROR_VARIABLE  BEAUTIFUL_ERROR
+)
```

Comment:
Could this use `add_custom_command()` instead so that CMake does not become even slower? (and so that this can be run in parallel to other things).
Something like the following might do:
```CMake
add_custom_target(OUTPUT docs/06-appearence.md
    COMMAND lua "${SOURCE_DIR}/docs/06-appearance.md.lua" "${BUILD_DIR}/docs/06-appearance.md")
```
(or one of the alternative forms of `add_custom_target()` that CMake has)